example/01_point_example.cpp

```diff
@@ -107,6 +107,33 @@ int main()
 
 
     // Some ways of getting point values
+    typedef model::point<double, 3, cs::cartesian> point_t;
+    typedef model::ring<point_t> ring_t;
+    typedef model::PolyhedralSurface<ring_t> poly_t;
+
+    poly_t polyhedron1;
+    poly_t polyhedron2 = {{{0,0,0}, {0, 1, 0}, {1, 1, 0}, {1, 0, 0}, {0, 0, 0}}, {{0, 0, 0}, {0, 1, 0}, {0, 1, 1}, {0, 0, 1}, {0, 0, 0}},
+    {{0, 0, 0}, {1, 0, 0}, {1, 0, 1}, {0, 0, 1}, {0, 0, 0}}, {{1, 1, 1}, {1, 0, 1}, {0, 0, 1}, {0, 1, 1}, {1, 1, 1}}, {{1, 1, 1}, {1, 0, 1}, {1, 0, 0}, {1, 1, 0}, {1, 1, 1}}, 
+    {{1, 1, 1}, {1, 1, 0}, {0, 1, 0}, {0, 1, 1}, {1, 1, 1}} };
+    
+    std::cout << wkt(polyhedron2) << std::endl;
+
+    //append(polyhedron1[0], point_t{1, 0, 0});
+    //append(polyhedron1[0], point_t{0, 0, 0});
+    //append(polyhedron1[0], point_t{0, 1, 0});
+    //append(polyhedron1[0], point_t{1, 1, 0});
+    //append(polyhedron1[0], point_t{0, 0, 0});
+    read_wkt("POLYHEDRALSURFACE(((0 0 0, 0 1 0, 1 1 0, 1 0 0, 0 0 0)),((0 0 0, 0 1 0, 0 1 1, 0 0 1, 0 0 0)),((0 0 0, 1 0 0, 1 0 1, 0 0 1, 0 0 0)),((1 1 1, 1 0 1, 0 0 1, 0 1 1, 1 1 1)),((1 1 1, 1 0 1, 1 0 0, 1 1 0, 1 1 1)),((1 1 1, 1 1 0, 0 1 0, 0 1 1, 1 1 1)))", polyhedron1);
+
+    std::cout << wkt(polyhedron1) << std::endl;
+
```

Comment:
The added code below looks like a small personal test.
At any rate, it doesn't belong to point / polyhedron and can just be deleted, I think.